modules/linear_boltzmann_solvers/lbs_solver/lbs_solver.cc

```diff
@@ -463,6 +463,8 @@ LBSSolver::OptionsBlock()
   params.AddOptionalParameter(
     "max_ags_iterations", 100, "Maximum number of across-groupset iterations.");
   params.AddOptionalParameter("ags_tolerance", 1.0e-6, "Across-groupset iterations tolerance.");
+  params.AddOptionalParameter(
+    "ags_pointwise_convergence", false, "Flag that enables pointwise convergence checking.");
```

Comment:
It may be better to have this be a string constrained to "ptwise" and "l2" (with default "l2" in this case) to make it clear what the convergence check is when specifying it in the input file.

Reply:
Done.